### PR TITLE
Create accessories

### PR DESCRIPTION
**Pull request by QuanLuu97**

### CODE REVIEW

**Comment by QuanLuu97** [comment]
ready

File: project1/resources/views/admin/accessories/edit.blade.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        </div>
+        <div class="col-md-6">
+            <div class="card">
+                <div class="card-header">{{ __('accessories.edit') }}</div>
+                <div class="card-body">
+                    {!! Form::open([ 'route' => ['update-accessories', $accessory->id] ]) !!}
+                        <div class="form-group">
+                            <div class="row">
+                                <label for="name">{{ __('accessories.name') }}</label>
+                                <input type="name" name="name" value="{{ $accessory->name }}" class="form-control">
```
Comment (BunnyPi04) [inline project1/resources/views/admin/accessories/edit.blade.php]: sử dụng laravel collective

File: project1/resources/views/admin/accessories/create.blade.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                        </div>
+                        <div class="form-group">
+                            <div class="row">
+                                <label for="content">{{ __('accessories.content') }}</label>
+                                <textarea name="content" class="ckeditor"></textarea>
+                            </div>
+                        </div>
+                        <div class="form-group">
+                            <div class="row">
+                                <input type="submit" value="{{ __('accessories.submit') }}" class="btn btn-success mr-2">
```
Comment (BunnyPi04) [inline project1/resources/views/admin/accessories/create.blade.php]: sử dụng laravel collective

**Comment by BunnyPi04** [comment]
commented

**Pull request closed without merging**